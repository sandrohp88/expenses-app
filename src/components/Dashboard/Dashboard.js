import React from 'react'
import ExpenseList from '../ExpenseList/ExpenseList'
import ExpensesFilterInput from '../ExpensesFilterInput/ExpensesFilterInput'
import ExpensesSummary from '../ExpensesSummary/ExpensesSummary'
const Dashboard = () => {
  
  return (
    <div>
      {/* <h1>Dashboard</h1> */}
      <ExpensesSummary />
      <ExpensesFilterInput />
      <ExpenseList />
    </div>
  )
}

export { Dashboard }
