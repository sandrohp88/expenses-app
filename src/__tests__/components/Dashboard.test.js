import { shallow } from 'enzyme'
import React from 'react'
import { Dashboard } from '../../components/Dashboard/Dashboard'

test('render Dashboard without crash', () => {
  const wrapper = shallow(<Dashboard />)
  expect(wrapper).toMatchSnapshot()
})
