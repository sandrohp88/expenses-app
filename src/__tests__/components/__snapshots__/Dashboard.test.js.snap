// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Dashboard without crash 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
  <Connect(ExpensesFilterInput) />
  <Connect(ExpensesSummary) />
  <Connect(ExpenseList) />
</div>
`;
